src/components/atoms/input/Radio/Radio.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { HTMLAttributes, useState } from "react";
+
+export interface RadioProps extends HTMLAttributes<HTMLElement> {
+    id?: string;
+    className?: string;
+    value: string;
+    checked: boolean;
+    type: string;
+    onClick: () => void;
+}
+
+const Radio: React.FC<RadioProps> = (props) => {
+    const [value, setValue] = useState("")
+
+    const buttonClicked = (e) => {
+        setValue(e.target.value)
+        console.log("Clicked")
```

Comment:
youll want to make sure you pull these out before you push to the repo, its useful for dev but we dont want logs getting into the codebase